pkg/oras/pull.go

```diff
@@ -4,12 +4,12 @@ import (
 	"context"
 	"sync"
 
-	orascontent "github.com/deislabs/oras/pkg/content"
-
 	"github.com/containerd/containerd/content"
 	"github.com/containerd/containerd/images"
 	"github.com/containerd/containerd/log"
 	"github.com/containerd/containerd/remotes"
+	orascontent "github.com/deislabs/oras/pkg/content"
```

Comment:
I think we removed all deislabs referencs and should be under `github.com/oras-project/oras/`

Reply:
https://github.com/oras-project/oras/blob/artifacts/go.mod -- it's still desilabs in the artifacts branch